charlesinto/iReporter
# #162590567 Update intervention record

#### What does this PR do?

- Update intervention comment

#### Description of Task to be completed?
- implement jwt authentication
- verify user token
- user should be able to update comment

#### How should this be manually tested?
- Clone the repo - `git clone  git repo address`
- Enter into the project directory - `cd repo folder`
- switch to this branch - `PR branch`
-  Start  the frontend.
- Navigate to `endpoint`
- Open `Postman`
- Grab your token from the application tab in Developers tab
- On postman add the token in headers
- - On `postman` Send a `PATCH` request to `http://localhost:7000/api/v1/interventions/:id/comment`
- You should receive a successful response

#### Any background context you want to provide?
Request should be structure like:

`{comment: "string"}`

`

#### What are the relevant pivotal tracker stories?
[#162590567](https://www.pivotaltracker.com/story/show/162590567)

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/26615085/49829698-57ccf280-fd8f-11e8-9549-e36d0f524bb1.png)





#### Questions:
N/A
